cmd/vault-secret-collection-manager/main.go

```diff
@@ -47,7 +47,9 @@ func parseOptions() (*option, error) {
 }
 
 func main() {
+	version.Name = "vault-secret-collection-manager"
 	logrusutil.ComponentInit()
+	logrus.SetLevel(logrus.DebugLevel)
```

Comment:
Do we not have a default flag for this?

Reply:
I don't think so, in the prow components this comes from the config